kratos/geometries/geometry.h

```diff
@@ -1724,6 +1724,28 @@ class Geometry
         return rResult;
     }
 
+    /**
+    * @brief This method maps from dimension space to working space and computes the
+    *        number of derivatives at the dimension parameter.
+    * @param LocalCoordinates The local coordinates in dimension space
+    * @param Derivative Number of computed derivatives
+    * @return std::vector<array_1d<double, 3>> with the coordinates in working space
+    * @see PointLocalCoordinates
+    */
+    virtual void GlobalDerivatives(
+        std::vector<CoordinatesArrayType>& rGlobalCoordinates,
```

Comment:
The order in which the derivatives are stored should be explained in the comments.

Reply:
Thanks for the comment. I put it accordingly.

Reply:
I think in #4777 we agreed on the interface such function should have...